src/runtime_src/core/common/module_loader.cpp

```diff
@@ -146,6 +147,27 @@ shim_path()
   return path;
 }
 
+static std::vector<std::string>
+driver_plugin_paths()
+{
+  std::vector<std::string> ret;
+  bfs::directory_iterator p{shim_path().parent_path()};
+
+  // All driver plug-ins are in the same directory as shim .so and with below prefix and suffix.
+  const std::string pre = "libxrt_driver_";
+  const std::string suf = std::string(".so.") + XRT_VERSION_MAJOR;
+  while (p != bfs::directory_iterator{}) {
+    std::string name = p->path().filename().string();
```

Comment:
This can be const and auto since the type is obvious.
```suggestion
    const auto name = p->path().filename().string();
```